frontend_lib/src/component/Button/IconButton.styl

```
@@ -72,7 +72,7 @@
       &:hover
         background-color lightGrey2
 
-@media (min-width: min-xs) && (max-width: max-xs)
+@media (min-width: min-sm) && (max-width: max-sm)
```

If you want the rule to apply to `xs` AND `sm` screens, the rule must be written in one of the two ways below. You can choose. If not, you can ignore this comment and resolve the discussion.

```suggestion
@media (min-width: min-xs) && (max-width: max-sm)
```

```suggestion
@media (max-width: max-sm)
```